frontend/app/model/Nav.scala

```diff
@@ -10,6 +10,8 @@ object Nav {
     subNav: Seq[NavItem] = Nil
   )
 
+  case class NavAnchor(href: String, title: String)
```

Comment:
Not sure I agree, it's common to the other Navigation code in here, given it's a small case class I don't quite see the benefit of creating something dedicated in the `views` package for it.